i18n/GLOSSARY_fr.md

```diff
@@ -19,6 +19,13 @@ Ce glossaire accompagne les [principes GitOps](./PRINCIPLES_fr.md), et autres do
 
     Lorsque l'état réel d'un système s'est éloigné ou est en train de s'éloigner de [l'état souhaité](#etat-souhaite), on parle alors de dérive.
 
+- ## Pull
+
+    [Principe 3](./PRINCIPLES_fr.md) ("Pull" Automatiquement) spécifie que l'état souhaité doit être extrait ("pull") plutôt que "push", principalement parce que les agents logiciels doivent être capables d'acceder l'[état souhaité](#etat-souhaite) depuis le [stockage d'état](#stockage-d-etat) à _tout_ moment, et pas seulement lorsqu'il y a un changement intentionnel dans le stockage d'état déclenchant un événement de push.
```

Comment:
"poussé" instead of "push"
Anchor  are  (#état-souhaité) and (#stockage-détat) (quote is ignored) and (#réconciliation)

My full translation was 

>   [Principe 3](./PRINCIPLES_fr.md) (Extraction automatique) indique que l'état souhaité doit être extrait plutôt que poussé, principalement parce que les agents logiciels doivent être capable d'accéder à tout moment à [l'état souhaité](#état-souhaité) dans le [stockage d'état](#stockage-détat), pas seulement lors d'un changement intentionel dans le stockage d'état déclanchant un évėnement de type "push". C'est une condition préalable pour que la réconciliation se produise en continu, comme spécifié dans le [principe 4](./PRINCIPLES_fr.md) (Reconcilié en permanence). Notez que, contrairement au CI/CD traditionnel, où l'automatisation est généralement pilotée par des déclencheurs prédéfinis, dans GitOps, la réconciliation est déclenchée chaque fois qu'il y a une divergence. La divergence peut être due à l'état réel qui s'éloigne involontairement des déclarations d'état souhaitées, et pas seulement suite à une nouvelle version de déclaration d'état souhaitée modifiée intentionnellement.